.yarnrc.yml

```diff
@@ -25,3 +25,4 @@ plugins:
     spec: "https://yarn.build/latest"
 
 yarnPath: .yarn/releases/yarn-3.2.3.cjs
+enableScripts: false
```

Comment:
What is this?

Reply:
It was auto generated by Lavamoat

Reply:
Allows only authorized lifecycle hooks to run. All auto generated

Reply:
> Allows only authorized lifecycle hooks to run. All auto generated

you can't simply arbitrarily disable all pre-post scripts.